src/actions/prListAction.js

```diff
@@ -0,0 +1,24 @@
+import { FETCH_PRLIST } from './actionTypes';
+import axios from 'axios';
+
+const apiUrl = 'http://localhost:5010/tracker/list';
+
+export const fetchPRs = (prs) => {
+    return {
+      type: FETCH_PRLIST,
+      prs
+    }
+  };
+  
+  export const fetchAllPRs = () => {
+    return (dispatch) => {
```

Comment:
As we are getting data from hitting the api so we set the loader here which will true when data is loading and set false as get the data.